job_definitions/notify_suppliers_of_awarded_briefs.yml

```diff
@@ -0,0 +1,52 @@
+{% set environments = ['preview', 'production'] %}
+{% set NOTIFY_TEMPLATE_ID = '7379f74f-3f51-4b50-950c-d86b7880e999' %}
+---
+{% for environment in environments %}
+- job:
+    name: "notify-suppliers-of-awarded-briefs-{{ environment }}"
+    display-name: "Notify suppliers that a brief they applied to has been awarded - {{ environment }}"
+    project-type: freestyle
+    description: "Send email notifications to supplier users if they have a completed application to a brief that has been awarded."
+    parameters:
+      - string:
+          name: AWARDED_AT
+          default: null
+          description: "Specify a given date, send emails for briefs awarded on that date (format: YYYY-MM-DD)"
+      - string:
+          name: BRIEF_RESPONSE_IDS
+          default: null
+          description: "List of BriefResponse IDs to be emailed (e.g. following a failure)"
+      - bool:
+          name: DRY_RUN
+          default: false
+          description: "List notifications that would be sent without sending the emails"
+    triggers:
+      - timed: "H 8 * * *"
+    publishers:
+      - trigger-parameterized-builds:
+          - project: notify-slack
+            condition: UNSTABLE_OR_WORSE
+            predefined-parameters: |
+              USERNAME=supplier-brief-awards
+              JOB=Notify suppliers of brief awards {{ environment }}
+              ICON=:crystal_ball:
+              STAGE={{ environment }}
+              STATUS=FAILED
+              URL=<${BUILD_URL}consoleFull|#${BUILD_NUMBER}>
+              CHANNEL=#dm-release
+    builders:
+      - shell: |
+          if [ -n "AWARDED_AT" ]; then
```

Comment:
I think you're checking here that the string `"AWARDED_AT"` has a non zero length, which it always will. Should it be `[ -n "$AWARDED_AT" ]`?

Reply:
Whoops - good spot! Will fix all these.